### ISSUE TITLE
Add choice to save raw files even if a MIME hook is used

### ISSUE THREAD

**Issue by isoraqathedh** [skyjake/lagrange#338]
Currently, if a MIME hook is used to render an otherwise unknown file type, the displayed file would the be the target for saving. While this is useful in some situations, in others, it's more reasonable to have the original, untranslated file downloaded.

I propose that the original response, minus the Gemini first-line response, be available to save to disk as well as the final rendered output.

This would be useful for one of the things I would like to do with hooks, specifically being able to specify a "last-resort" translation option.

**Comment by skyjake** [comment]
Yeah that makes sense. At least there should be a feature to re-fetch the page with MIME hooks disabled.

Note that currently if you download a file via the link context menu ("Download Linked File"), MIME hooks are bypassed and the original data is saved to downloads.
